<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>link_mycroft</class>
 <widget class="QWidget" name="link_mycroft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UBUVoiceAssistant</string>
  </property>
  <widget class="QLabel" name="lblWelcome">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>#WelcomeLabel</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblFirstRun">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>#FirstRunExplanation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imgAddDevice">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>191</width>
     <height>301</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../imgs/mycroft_add_device.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrev">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>prev</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNext">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>560</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRegisterAddDevice">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string>#lblregister</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="imgSelectVoice">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>531</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../imgs/mycroft_set_voice.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelectVoice">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>#selectVoiceEngine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="picInputCode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>591</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../imgs/mycroft_input_code.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfoCode">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>#inputcode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblCode">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>510</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>#code</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <zorder>imgSelectVoice</zorder>
  <zorder>lblWelcome</zorder>
  <zorder>lblFirstRun</zorder>
  <zorder>imgAddDevice</zorder>
  <zorder>btnPrev</zorder>
  <zorder>btnNext</zorder>
  <zorder>lblRegisterAddDevice</zorder>
  <zorder>lblSelectVoice</zorder>
  <zorder>picInputCode</zorder>
  <zorder>lblInfoCode</zorder>
  <zorder>lblCode</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
